ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -122,6 +122,7 @@ public static void setUp() throws IOException {
 
         ConnectionStringBuilder engineCsb = ConnectionStringBuilder.createWithAadApplicationCredentials(System.getenv("ENGINE_CONNECTION_STRING"), appId,
                 appKey, tenantId);
+        engineCsb.setUserNameForTracing("Java_E2ETest_ø");
```

Comment:
This is great for testing that sending a non-ASCII char doesn't cause ingestion to **fail**. But can we use a getter to ensure in this test that it properly translates the char to a "?"?

Reply:
Discussed offline